### ISSUE TITLE
Errors while installing

### ISSUE THREAD

**Issue by megadaabiss** [idiap/IBDiarization#20]
This error appears when I try to run the script for pydiarization after installing and configuring all binaries required.
![Screenshot from 2021-05-28 18-48-43](https://user-images.githubusercontent.com/56893500/120018588-aadea280-bfe7-11eb-893c-c53854c75121.png)
And when I try to open the config file of compute-mfcc-feats to see anything wrong, this comes as an output:
![Screenshot from 2021-05-28 18-48-24](https://user-images.githubusercontent.com/56893500/120018782-e8433000-bfe7-11eb-979f-c392d77426e5.png)
Please answer as soon as possible because I extremely need this library to work for my project, THANKS !

**Comment by wdroz** [comment]
There is an error while invoking compute-mfcc-feats. You can see in the text of the error that the wav frequency expected doesn't match the one in the config. If you don't want to edit the code, you must resample the wav to match the expected frequency.

**Comment by megadaabiss** [comment]
I don't think that the difference in frequency is the main problem, because it just gave a warning on this issue but continued working. There's an another problem which I can't figure out till now. And if it's, where can I write the "--allow -downsample=true" ? Thanks for replying !

**Comment by wdroz** [comment]
The warning is about the channel, not the sample frequency mismatch. Pydiarization doesn't halt the program on errors.

**Comment by megadaabiss** [comment]
you're right, the problem was in the frequency, thanks for the help! 
But I got an another question, what is the accuracy of the rttm file ?? Because I guess it isn't high from the output that is coming for me.
